x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/unified_inventory.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiDataGridSorting } from '@elastic/eui';
+import React from 'react';
+import useEffectOnce from 'react-use/lib/useEffectOnce';
+import {
+  type EntityColumnIds,
+  type EntityType,
+  entityPaginationRt,
+} from '../../../common/entities';
+import { EntitiesGrid } from '../entities_grid';
+import { useInventoryAbortableAsync } from '../../hooks/use_inventory_abortable_async';
+import { useInventoryParams } from '../../hooks/use_inventory_params';
+import { useInventoryRouter } from '../../hooks/use_inventory_router';
+import { useKibana } from '../../hooks/use_kibana';
+import { useInventorySearchBarContext } from '../../context/inventory_search_bar_context_provider';
+import { InventorySummary } from './inventory_summary';
+import { extractPaginationParameter } from '../../utils/extract_pagination_parameter';
+
+export function UnifiedInventory() {
+  const {
+    services: { inventoryAPIClient },
+  } = useKibana();
+  const { refreshSubject$ } = useInventorySearchBarContext();
+  const { query } = useInventoryParams('/');
+  const { sortDirection, sortField, kuery, entityTypes, pagination: paginationQuery } = query;
+  const inventoryRoute = useInventoryRouter();
+  const pagination = extractPaginationParameter(paginationQuery);
```

Comment:
in case this function throws an exception, how are we going to handle it in the UI?

Reply:
My thought is that maybe we should reset the query state, so basically remove the query params and refresh the page? Problematic query params are more easily resolved by... just getting rid of them.

Reply:
yeah, we just need to ensure this doesn't crash the page.